test/unit/org/apache/cassandra/net/WriteCallbackInfoTest.java

```diff
@@ -32,19 +34,35 @@
 import org.apache.cassandra.locator.InetAddressAndPort;
 import org.apache.cassandra.schema.MockSchema;
 import org.apache.cassandra.schema.TableMetadata;
+import org.apache.cassandra.service.StorageService;
 import org.apache.cassandra.service.paxos.Commit;
 import org.apache.cassandra.utils.ByteBufferUtil;
 
 import static org.apache.cassandra.locator.ReplicaUtils.full;
 
 public class WriteCallbackInfoTest
 {
+    private InetAddressAndPort testEp;
+
     @BeforeClass
     public static void initDD()
     {
         DatabaseDescriptor.daemonInitialization();
     }
 
+    @Before
+    public void setup() throws Exception
+    {
+        testEp = InetAddressAndPort.getByName("192.168.1.1");
+        StorageService.instance.getTokenMetadata().updateHostId(UUID.randomUUID(), testEp);
```

Comment:
nice one

Reply:
Thank you @smiklosovic. Are you reviewing the PR?